config/database.yml

```diff
@@ -27,3 +27,4 @@ test:
 production:
   <<: *default
   database: db/production
+  username: vagrant
```

Comment:
This is fine for now, but for future information, this should either be something more generic (like a "deploy" user), or even better, and environment variable